src/resources/packages/classy/types/LocalizedData.d.ts

```diff
@@ -74,6 +53,7 @@ export type CurrencySettings = {
  */
 export type Settings = {
 	currency: CurrencySettings;
+	startOfWeek: number;
```

Comment:
Should this be made into an interval of intervals from `0` to `6`?

Reply:
Yes, good catch. I think we have a type in Common for that. I'll switch to that type.